### PR TITLE
Dark Mode Dialog css fixes

### PR DESCRIPTION
**Pull request by Andreas-Kainz**
input elements use css var's

| before | after |
| --------- | -------- |
| Screenshot_20230428_231244 | Screenshot_20230428_234537 |
| Screenshot_20230428_234608 | Screenshot_20230428_231314

Change-Id: Ie2ebe22bcf7fc377894f4b43c1bb081a3d5f5c16

### CODE REVIEW

**Comment by Andreas-Kainz** [comment]
| before | after |
| ---------- | ------- |
| Screenshot_20230429_000957 | Screenshot_20230429_002221 |
| Screenshot_20230429_001603 | Screenshot_20230429_002209 |

**Comment by Andreas-Kainz** [comment]
| before | after |
| --------- | -------- |
| Screenshot_20230429_004131 | Screenshot_20230429_004010 |

File: browser/css/btns.css [hunk 1]
```diff
@@ -64,12 +64,13 @@ button:not(.ui-tab):not(.ui-corner-all):not(.button-primary):not(.unobutton) {
...
 	color: var(--color-text-lighter) !important;
-	background-color: white !important;
+	background-color: var(--color-background-lighter) !important;
+	border: 1px solid var(--color-border-lighter) !important;
 }
 
 .ui-pushbutton[disabled]:hover {
 	cursor: not-allowed !important;
-	background: white !important;
+	background: var(--color-background-darker) !important;
```
Comment (pedropintosilva) [inline browser/css/btns.css:73]: Follow up (not a blocker for this PR) the background here( in .ui-pushbutton[disabled]:hover) is not bieng applied  due to rules from `.button-secondary:hover, button:not(.ui-corner-all):not(.button-primary):not(.unobutton):hover`

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
just please add the missing `color` to the var name and we can merge it : )

File: browser/css/btns.css (hunk 1 above)
Comment (Andreas-Kainz) [reply browser/css/btns.css:73]: To be honest, I don't know that much about the css rules at device-desktop.css so I can't change there that much (without destroy something).

What I don't understood is, that there are so much rules with `!important` elements.

**Review by pedropintosilva — APPROVED** [review]
Thanks! (note:next time better to rebase and amend the existing initial commit instead)

**Pull request merged**